.travis.yml

```diff
@@ -10,9 +10,10 @@ rust:
 - beta
 - nightly
 
-branches:
-  only:
-    - master
+# branches:
```

Comment:
Should we match the Appveyor config and exclude `gh-pages` here? Also I _think_ this was origionally to stop us getting two builds for pull requests. It doesn't seem to be needed any more though.